dotnet/src/webdriver/StackTraceElement.cs

```diff
@@ -43,18 +45,18 @@ public StackTraceElement()
         /// Initializes a new instance of the <see cref="StackTraceElement"/> class using the given property values.
         /// </summary>
         /// <param name="elementAttributes">A <see cref="Dictionary{K, V}"/> containing the names and values for the properties of this <see cref="StackTraceElement"/>.</param>
-        public StackTraceElement(Dictionary<string, object> elementAttributes)
+        public StackTraceElement(Dictionary<string, object>? elementAttributes)
         {
             if (elementAttributes != null)
             {
                 if (elementAttributes.ContainsKey("className") && elementAttributes["className"] != null)
                 {
-                    this.className = elementAttributes["className"].ToString();
+                    this.className = elementAttributes["className"].ToString() ?? "";
```

Comment:
It is reasonable to convert it to: `TryGetValue(...)` and assign if not `null`. Otherwise do nothing, because of field is already initializaed with defauls in ctor.

Reply:
I converted `StackTraceElement` and `ErrorResponse` to use `TryGetValue(...)`, it looks much better now.